app/components/add-layer/template.hbs

```diff
@@ -1,19 +1,25 @@
 {{#liquid-if isOpen use='toDown'}}
     <div class="add-layer {{if isOpen 'open'}}">
         <div class="container">
+          
             <div class="row p-v-md">
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Wiki</button>
+                
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'wiki'}}>{{fa-icon "wikipedia-w"}} <br /> Add Wiki layer </button>
                 </div>
-                <div class="col-sm-2">
-                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Link</button>
+                <div class="col-xs-2">
+                    <button class="btn btn-default btn-block" {{action 'addLayer' 'link'}}>{{fa-icon "link"}} <br /> Add Link layer</button>
```

Comment:
Do we need Add and layer in the language? I removed them because I thought they were not strictly necessary. If we keep 'add' then 'layer' doesn't need to be there. It's more of our internal name for it anyway.